antarest/study/storage/rawstudy/model/filesystem/root/user/expansion/settings.py

```diff
@@ -59,6 +59,7 @@ def __init__(self, context: ContextServer, config: FileStudyTreeConfig):
         if self.config.version > 800:
             types["relative-gap"] = float
             types["solver"] = str
+            types["batch-size"] = int
```

Comment:
Please, also add `batch-size:int = 0` in the docstring.

Reply:
I'm not sure for `>= 800` I think it should stay `> 800` because i see this in the xpansion documentation https://antares-xpansion.readthedocs.io/en/stable/CHANGELOG/#features_5)

    Antares-Xpansion is now compatible with Antares v8.1 studies
    Add relative_gap option as a stopping criterion for the Antares-Xpansion algorithm.


I will add the docstring and you can merge after my commit